src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ToolStripActionList.cs

```diff
@@ -38,7 +35,7 @@ private bool CanAddItems
         get
         {
             // Make sure the component is not being inherited -- we can't delete these!
-            InheritanceAttribute ia = (InheritanceAttribute)TypeDescriptor.GetAttributes(_toolStrip)[typeof(InheritanceAttribute)];
+            InheritanceAttribute? ia = (InheritanceAttribute?)TypeDescriptor.GetAttributes(_toolStrip)[typeof(InheritanceAttribute)];
             if (ia is null || ia.InheritanceLevel == InheritanceLevel.NotInherited)
```

Comment:
Maybe we can simplify this down to
```suggestion
            if (!TypeDescriptorHelper.TryGetAttribute(_toolStrip, out InheritanceAttribute? ia) || ia.InheritanceLevel == InheritanceLevel.NotInherited)
```
? Same thing on line 53

Reply:
Sure.